src/systems/a320_systems/src/hydraulic/flaps_computer.rs

```diff
@@ -124,21 +112,21 @@ impl SlatFlapControlComputer {
         match flap_conf {
             FlapsConf::Conf0 => Angle::new::<degree>(0.),
             FlapsConf::Conf1 => Angle::new::<degree>(0.),
-            FlapsConf::Conf1F => Angle::new::<degree>(10.),
-            FlapsConf::Conf2 => Angle::new::<degree>(15.),
-            FlapsConf::Conf3 => Angle::new::<degree>(20.),
-            FlapsConf::ConfFull => Angle::new::<degree>(40.),
+            FlapsConf::Conf1F => Angle::new::<degree>(120.22),
+            FlapsConf::Conf2 => Angle::new::<degree>(145.51),
+            FlapsConf::Conf3 => Angle::new::<degree>(168.35),
+            FlapsConf::ConfFull => Angle::new::<degree>(251.97),
```

Comment:
Perhaps we could add a comment that these are not actual flap/slat angles, but rather the angles coming from the FPPU?

Reply:
Done thanks!